corehq/apps/userreports/expressions/specs.py

```diff
@@ -258,14 +258,17 @@ class EvalExpressionSpec(JsonObject):
     type = TypeProperty('evaluator')
     statement = StringProperty(required=True)
     context_variables = DictProperty(required=True)
+    datatype = DataTypeProperty(required=False)
 
     def configure(self, context_variables):
         self._context_variables = context_variables
 
     def __call__(self, item, context=None):
         var_dict = self.get_variables(item, context)
         try:
-            return eval_statements(self.statement, var_dict)
+            untransformed_value = eval_statements(self.statement, var_dict)
+            transform = transform_from_datatype(self.datatype)
+            return transform(untransformed_value) if transform else untransformed_value
         except (InvalidExpression, SyntaxError, TypeError, ZeroDivisionError):
             return None
```

Comment:
What's the convention/contract here? If there's something wrong with the expression then return `None`?